src/game/client/draw/drawable.h

```diff
@@ -167,6 +167,15 @@ class Drawable : public Thing, public SnapShot
         float m_totalZ;
     };
 
+    enum DrawBits
+    {
+        DRAW_BIT_DISABLED = 1,
```

Comment:
these are based on?

Reply:
The places that I saw calling clearDrawBit and setDrawBit.